R/obic.R

```diff
@@ -338,12 +344,22 @@ obic_field <- function(B_SOILTYPE_AGR,B_GWL_CLASS,B_SC_WENR,B_HELP_WENR,B_AER_CB
     # Calculate indicators for soil biological functions
     dt[, I_B_DI := ind_resistance(A_SOM_LOI)]
     dt[, I_B_SF := ind_pmn(D_PMN)]
-  
-    # Calculate indicators for groundwater functions
-    # dt[, I_H_GWR := ind_gw_recharge(B_LU_BRP, D_PSP, D_WRI_K, I_P_SE, I_P_CO, B_DRAIN, B_GWL_CLASS)]
-    # dt[, I_H_NGW := ind_n_efficiency(D_NLEACH_GW,'gw')]
-    # dt[, I_H_NSW := ind_n_efficiency(D_NLEACH_GW,'sw')]
-    # dt[, I_H_PEST := ind_pesticide_leaching(D_PESTICIDE)]
+    
+    if(useClassicOBI == FALSE){
+      # Calculate indicators for groundwater functions
+      dt[, I_H_NGW := ind_n_efficiency(D_NLEACH_GW,'gw')]
+      dt[, I_H_NSW := ind_n_efficiency(D_NLEACH_GW,'sw')]
```

Comment:
@KeesDoolNMI Here the indicators for groundwater and surface water are calculated but they both use the D_ value for groundwater whilst there is a D_ value for surface water. I guess that this is not done on purpose, correct?